schemas/development/openservice.schema.json

```diff
@@ -27,15 +27,10 @@
                   "enum": ["OpenService"]
                 },
                 "Service": {
```

Comment:
I would have expected an approach more to eg. `WithTrafficStats` or `WithDestination`.

Reply:
I started implementing the `Service` shared type and would like to propose that. Do you know of a way to add a MR to a MR xD?

Reply:
# Example
xarf_shared 
```json
"Service": {
    "type": "object",
    "description": "Specify a service to report",
    "required": ["ServiceName"],
    "properties": {
        "ServiceName": {
            "type": "string"
         },
         "ServiceVersion": {
             "type": "string",
             "description": "Affected service version or version range"
         }
    }
},
"TransportProtocol": {
    "type": "object",
    "description": "Transport protocol used by the reportee",
    "properties": {
        "TransportProtocol": {
            "type": "string",
            "enum": ["tcp", "udp"]
        }
    }
}
```

Usage
```json
{
    "$ref": "xarf_shared.schema.json#/TransportProtocol"
},
{
    "$ref": "xarf_shared.schema.json#/Service"
}
```